test/login.e2e-spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as faker from 'faker';
+import { StatusCodes } from 'http-status-codes';
```

Comment:
I suggest you to use nestjs `HttpStatus`.

Reply:
Resolved on 1d84f9a3eaf92a35983aebc14843850a8cea13ab and 7ff3f7ee216842cd7af78536a1ce255a2224db7a.